### PR TITLE
Fix project to add icons and use cdap-maven-plugin for building

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: kafka-plugins-0.8/pom.xml [hunk 1]
```diff
@@ -51,6 +52,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```
Comment (anwar6953) [inline kafka-plugins-0.8/pom.xml]: just use the non-SNAPSHOT version?
Is the SNAPSHOT version even still available?

Comment (bdmogal) [reply kafka-plugins-0.8/pom.xml]: it seems to compile fine. I'll update though

Comment (bdmogal) [reply kafka-plugins-0.8/pom.xml]: or maybe its my local

Comment (anwar6953) [reply kafka-plugins-0.8/pom.xml]: ya, I suspect you have the dependent jar cached locally.

File: kafka-plugins-0.8/pom.xml [hunk 2]
```diff
@@ -51,6 +52,25 @@
...
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```
Comment (yaojiefeng) [inline kafka-plugins-0.8/pom.xml]: Does this need to be 9.0.0-SNAPSHOT or 6.0.0-SNAPSHOT?

File: kafka-plugins-0.8/pom.xml (hunk 1 above)
Comment (bdmogal) [reply kafka-plugins-0.8/pom.xml]: fixed

File: kafka-plugins-0.9/pom.xml [hunk 3]
```diff
@@ -52,6 +53,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```
Comment (yaojiefeng) [inline kafka-plugins-0.9/pom.xml]: Does this need to be 9.0.0-SNAPSHOT or 6.0.0-SNAPSHOT?

File: kafka-plugins-0.8/pom.xml [hunk 4]
```diff
@@ -51,6 +52,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```
Comment (yaojiefeng) [inline kafka-plugins-0.8/pom.xml]: Does this need to be 9.0.0-SNAPSHOT or 6.0.0-SNAPSHOT?

File: kafka-plugins-0.9/pom.xml [hunk 5]
```diff
@@ -52,6 +53,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```
Comment (yaojiefeng) [inline kafka-plugins-0.9/pom.xml]: Does this need to be 9.0.0-SNAPSHOT or 6.0.0-SNAPSHOT?

**Review by yaojiefeng — COMMENTED** [review]
Just a question about the version, otherwise LGTM

File: kafka-plugins-0.8/pom.xml (hunk 2 above)
Comment (bdmogal) [reply kafka-plugins-0.8/pom.xml]: fixed

File: kafka-plugins-0.9/pom.xml (hunk 3 above)
Comment (bdmogal) [reply kafka-plugins-0.9/pom.xml]: fixed

**Comment by bdmogal** [comment]
@yaojiefeng updated the version range. Please have another look.

**Comment by yaojiefeng** [comment]
LGTM

**Pull request merged**